test/optimizer/joins/get_cardinality_from_limit.test

```diff
@@ -0,0 +1,17 @@
+# name: test/optimizer/joins/get_cardinality_from_limit.test
+# description: when a limit is encountered, use the limit to know the cardinality
+# group: [joins]
+
+statement ok
+create table t_left as select (random() * 10000000)::INT a from range(40000);
+
+statement ok
+create table t_right as select range b from range(10000000);
+
+statement ok
+pragma explain_output='optimized_only'
+
+
```

Comment:
This needs a REGEX check, but the plans look very weird, so I don't know exactly what I want here

Reply:
If the plan looks like this:
```
┌───────────────────────────┐                             
│         PROJECTION        │                             
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                             
│             i             │                             
└─────────────┬─────────────┘                                                          
┌─────────────┴─────────────┐                             
│         HASH_JOIN         │                             
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                             
│           INNER           │                             
│           i = i           │                             
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ├──────────────┐              
│        Build Min: 0       │              │              
│        Build Max: 9       │              │              
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │              │              
│           EC: 10          │              │              
└─────────────┬─────────────┘              │                                           
┌─────────────┴─────────────┐┌─────────────┴─────────────┐
│         SEQ_SCAN          ││           LIMIT           │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││                           │
│             t0            ││                           │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││                           │
│             i             ││                           │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││                           │
│ Filters: i<=9 AND i IS NOT││                           │
│            NULL           ││                           │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││                           │
│           EC: 11          ││                           │
└───────────────────────────┘└─────────────┬─────────────┘                             
                             ┌─────────────┴─────────────┐
                             │         SEQ_SCAN          │
                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
                             │             t1            │
                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
                             │             i             │
                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
                             │           EC: 10          │
                             └───────────────────────────┘
```
You can probably do something like `.*SEQ_SCAN.*LIMIT.*` to check that the limit is on the rhs (build side).